extension-testcontainer/src/main/java/io.camunda.zeebe.process.test.extension.testcontainer/EngineContainer.java

```diff
@@ -6,6 +6,9 @@
 import org.testcontainers.containers.output.Slf4jLogConsumer;
 import org.testcontainers.utility.DockerImageName;
 
+/**
+ * Singleton object which manages access to the testcontainer that's running the test engine
+ */
```

Comment:
❓ Is this something the users interact with? If so, please provide an example of how to use it.

Reply:
It's not something for users to interact with.